dvc/info.py

```diff
@@ -136,11 +136,14 @@ def _get_supported_remotes():
 
 
 def get_fs_type(path):
-
-    partition = {
-        pathlib.Path(part.mountpoint): (part.fstype + " on " + part.device)
-        for part in psutil.disk_partitions(all=True)
-    }
+    partition = {}
+    for part in psutil.disk_partitions(all=True):
+        if part.fstype != "":
```

Comment:
Does this mean that psutil was not able to regognize fstype? If so, should we mark it as `unknown` then?

Reply:
Yes, in some cases I had an empty fstype, I haven't managed to reproduce the error since but I'm sure it exists. That's why I added this check just in case. And yes you are right! It's better to mark it as "unknown" if that's the case, I can fix that if you want?

Reply:
I see. Yeah, if `part.mountpoint` is present and meaningful, let's adjust that to `unknown`, should make more sense.